lib/metadata_json_deps/runner.rb

```diff
@@ -41,6 +50,7 @@ def run
             allMatch = false
             message += "\t#{dependency} (#{constraint}) *doesn't match* #{current}\n"
           end
+          message += "The dependency module #{dependency.upcase} is DEPRECATED.\n" if check_deprecated(current)
```

Comment:
Need a `\t` at the start of the string to ensure the message formatting is consistent.